[[ TSV ]] Use .tsv extension for tab delimited files

bwmilby opened the pull request:
Renamed tab separated text files with .tsv extension (property definitions/standalone settings)
Adjusted code (IDE Library) to load updated files
Formatted all files to a standard number of tabs to support Git formatting of tables
Moved field headings to the top of the file to support Git formatting of tables

montegoulding committed:
[[ TSV ]] Use .tsv extension for tab delimited files

bwmilby committed:
[[ TSV ]] Use .tsv extension for tab delimited files

Update .tsv files to contain the same number of tabs on each line.

bwmilby committed:
[[ TSV ]] Use .tsv extension for tab delimited files

Remove extra tabs

bwmilby committed:
[[ TSV ]] Use .tsv extension for tab delimited files

Remove extra tabs

bwmilby committed:
[[ TSV ]] Use .tsv extension for tab delimited files

Move field headers to the first line to support Git formatting.

montegoulding commented:
@livecode-vulcan review ok 17076c4

Thanks @bwmilby ! I'm sure this will make it much easier to read these files. I'm reviewing OK but will check with @livecodeali that he is happy for the headings to be above those lines at the top before merging. It does look a little odd but at least if you search you get the correct titles.

livecode-vulcan commented:
💙 review by @montegoulding ok 17076c47a6d6fd833b6d05022140480d55116463

bwmilby commented on Toolset/resources/supporting_files/standalone_settings/standaloneSettings.tsv:
```diff
@@ -1,131 +1,131 @@
 Brushes	Brushes	General	com.livecode.pi.boolean	true	false	Inclusion Options			Inclusions
-pdfPrinter	PDF Printer	General	com.livecode.pi.boolean	true	false	Inclusion Options		Inclusions
+pdfPrinter	PDF Printer	General	com.livecode.pi.boolean	true	false	Inclusion Options		Inclusions	
 scriptLibraries	Script Libraries	General	com.livecode.pi.set	true	false			execute:dispatch function "revSBLibraryList" to stack "revSBLibrary";get the result;sort it	Inclusions
```
Formatter doesn't like this line (13) (complains about illegal quoting)

bwmilby committed:
[[ TSV ]] Use .tsv extension for tab delimited files

standaloneSettings.tsv has an issue with the Git table view - line 13 has the " character which appears to be invalid
ios,includes and android,includes lines have an extra column inserted between property and label
this commit is to restore the extra column in those lines and add a tab to all other lines

bwmilby committed:
[[ TSV ]] Use .tsv extension for tab delimited files

Restore missing lines in standaloneSettings.tsv

livecodeali commented:
I'm fine with that, it was just an arbitrary marker really. The code in the IDE checks for `-----` at the beginning of the line to start reading property info, (as I'm sure @bwmilby figured out) so that's the only requirement without changing the IDE code.

montegoulding commented:
@livecode-vulcan review ok 4a4ff25

livecode-vulcan commented:
💙 review by @montegoulding ok 4a4ff251043704efb819bf0570e3f81e83780f71
